snap/quota/resources.go

```diff
@@ -262,6 +314,20 @@ func (qr *Resources) ValidateChange(newLimits Resources) error {
 		}
 	}
 
+	// Verify journal limits not being removed
```

Comment:
It's not clear to me why limits cannot be removed.

Reply:
This was a decision made that quota groups should be recreated with new limits instead of being able to remove limits. We could change this, but then we need to take a new decision about it